lsp/markdown/completion/retry.md

```diff
@@ -0,0 +1,9 @@
+Snippet for retrying an async operation.
```

Comment:
This should describe what it is, and more importantly where it is applicable, as it's only used in limited situations.